src/Webhooks.ts

```diff
@@ -139,15 +141,20 @@ export class Webhooks extends EventEmitter {
         try {
             let eventName: string|null = null;
             const body = req.body;
-            if (req.headers['x-hub-signature']) {
```

Comment:
We check for the a different header now, but Github [always sends both](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads).

Reply:
I think this is okay, the delivery header is always sent?

Reply:
Yeah, that's what I meant :)